skate-plugin/build.gradle.kts

```diff
@@ -86,6 +86,8 @@ tasks
 
 dependencies {
   implementation(libs.bugsnag)
+  implementation(libs.kotlin.compiler.embeddable)
+  implementation(libs.kotlin.stdlib)
```

Comment:
We should do compileOnly at most as IJ enforces its own bundled version, but still curious why you have to do this at all?

Reply:
[2c1c508](https://github.com/slackhq/slack-gradle-plugin/pull/552/commits/2c1c5089c55de533b229ee1b1481390387b86a51)